ee/management/commands/backfill_has_full_snapshot.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.sql.session_recording_events import SESSION_RECORDING_EVENTS_TABLE
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_REPLICATION
+
+
+class Command(BaseCommand):
+    help = "Backfill the has_full_snapshot materialized column on session_recording_events"
```

Comment:
Why do we need this?

Reply:
Fair question. It might be overkill, but the idea was that customers having issues could run it to see the benefits of the migration immediately instead of waiting 3-weeks.

I don't really understand how this backfill works or how much I/O it uses. I was following examples - and confirmed it works, but I'd love some guidance here. Other options seem to be:
* Add this backfill into the migration. Seems like the right choice if this backfill is not too computationally intensive.
* Don't migrate at all, and let the 3-week TTL run its course.
* Run the backfill as a background task (similar to how it happens on events + person properties).
* Add a command that runs the backfill. If customers are having trouble, they can run the command when it makes sense. Otherwise, it will catch up in 3-weeks. (my vote)

Reply:
IMO At best this will be run by 1-2 clients and will cause some maintanence overhead for a command that's really not needed for any fresh installations or most normal installations.

I'd instead save this as a one-off script to share with users who are having trouble under say posthog.com repo.